SHARDING.md

```diff
@@ -0,0 +1,254 @@
+# Sharding Playbook
+
+This playbook walks through the steps to create a new log shard.
+The steps should be the same for Rekor and the CT log, though the
+CT process for generating key material may differ.
+
+Roughly, the steps are:
+
+1. Create the GCP resources with Terraform
```

Comment:
same here: this seems specific to our public good instance.

Reply:
Yep, I'll move this playbook to PGI.